lib/action.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require_relative 'config'
+require 'active_support'
```

Comment:
You have `zeitwerk` installed, so could maybe use that.

Reply:
Sorry don't follow, how would that help? The method I am using here seems to be located in activesupport not Zeitwerk I think.

```ruby
require 'active_support/core_ext/string/filters'
```

Reply:
It is probably overkill, but Zeitwerk is an autoloader.  No problem with how you have done it.